app/workers/retirement_worker.rb

```diff
@@ -18,4 +19,13 @@ def deactivate_workflow!(workflow)
       Workflow.where(id: workflow.id).update_all(active: false)
     end
   end
+
+  def push_counters_to_event_stream(workflow)
```

Comment:
The `EventStream` class defaults to a UUID so that Kafka consumers always see the event_* fields, but leaves the option of overriding it (eg. deduping classifications).